### DISCUSSION TITLE
No suggestions shown

### DISCUSSION THREAD

**Discussion by wjadevries** [community/community#13954] [Copilot Conversations]
Just got an email that I have been invited to test Copilot.
Installed Copilot in VS Code and logged into GitHub, but nothing happens when I type the suggested example code function calculateDaysBetweenDate(begin, end) { in a JavaScript file.
Anything I need to do what's not in the Getting Started manual?

**Comment by PisanoScaff** [comment]
Same here!

**Comment by kennerstross** [comment]
Me too

**Comment by ChuTingzj** [comment]
+1

**Comment by Macguiwer** [comment]
+1
I´ve reinstalled copilot a few times.
even in admin mode.
I put a log.

GitHub Copilot 1.10.0.3 (v1.10.0.3@764d4f318)
GitHub Copilot Agent v1.10.0
[Error] An error occurred for 'CreateAsync': System.ArgumentException: No EditorOptionDefinition export found for the given option name: IntelliCodeGrayTextEnabledOption
Nombre del parámetro: optionId
   en Microsoft.VisualStudio.Text.EditorOptions.Implementation.EditorOptionsFactoryService.GetOptionDefinitionOrThrow(String optionId)
   en Microsoft.VisualStudio.Text.EditorOptions.Implementation.EditorOptions.SetOptionValue(String optionId, Object value)
   en Microsoft.VisualStudio.Internal.InlineSuggestions.Implementation.LegacySuggestionProvider.<SetEnabledOptionAsync>d__30.MoveNext() en D:\a\_work\1\s\src\InlineSuggestions\Implementation\LegacySuggestionProvider.cs:línea 155
--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---
   en System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en Microsoft.VisualStudio.Internal.InlineSuggestions.Implementation.LegacySuggestionProvider.<DisabledAsync>d__13.MoveNext() en D:\a\_work\1\s\src\InlineSuggestions\Implementation\LegacySuggestionProvider.cs:línea 66
--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---
   en System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en Microsoft.VisualStudio.Internal.InlineSuggestions.Implementation.SuggestionManager.<DisableProviderAsync>d__55.MoveNext() en D:\a\_work\1\s\src\InlineSuggestions\Implementation\SuggestionManager.cs:línea 285
--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---
   en System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en GitHub.Copilot.SuggestionManagerAdapter.<DisableProviderAsync>d__3.MoveNext()
--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---
   en System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en GitHub.Copilot.DisableIntellicodeHandler.<CreateAsync>d__4.MoveNext()
--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---
   en System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en GitHub.Copilot.CompositeSuggestionHandler.<InitializeAsync>d__11.MoveNext()
--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---
   en System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en GitHub.Copilot.CopilotSuggestionProvider.<CreateAsync>d__10.MoveNext()
--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---
   en System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en GitHub.Copilot.CopilotCommandHandler.<>c__DisplayClass36_0.<<GetProvider>b__2>d.MoveNext()
--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---
   en System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en Microsoft.VisualStudio.Threading.JoinableTask.CompleteOnCurrentThread()
   en Microsoft.VisualStudio.Threading.JoinableTask`1.CompleteOnCurrentThread()
   en GitHub.Copilot.CopilotCommandHandler.<>c__DisplayClass36_0.<GetProvider>b__0()
   en Microsoft.VisualStudio.Utilities.PropertyCollection.GetOrCreateSingletonProperty[T](Object key, Func`1 creator)
   en GitHub.Copilot.CopilotCommandHandler.GetProvider(

**Comment by saimnaveediqbal** [comment]
Had the same problem, the Copilot icon on the taskbar indicated that Copilot was working constantly. Suggestions started showing up after pressing ctrl + enter, which forced Copilot to synthesize suggestions I assume.

**Comment by coder4why** [comment]
Me too!

**Comment by zenetio** [comment]
Until now I never saw the icon suggestion in my VS Code. I only see the completion code while I am typing code or comments. Weird behavior!
